redhat_access_insights/connection.py

```diff
@@ -169,6 +153,13 @@ def get_proxies(self, config):
                 conf_proxy = scheme + location
             logger.debug("CONF Proxy: %s", conf_proxy)
             proxies = {"https": conf_proxy}
+
+        if os.environ.get('NO_PROXY') and conf_proxy:
+            logger.debug("Invalid Proxy configuration. "
+                         "You have environment variable NO_PROXY set "
+                         "as well as 'proxy' set in your configuration file. "
+                         "This is invalid. Do not expect normal results.")
```

Comment:
Wait nvm, the config file proxy setting is incompatible with NO_PROXY, the HTTPS_PROXY is compatible.

@lphiri what if we say "If you want to use NO_PROXY, use HTTPS_PROXY instead of the config file(/etc/path/to/file.conf) proxy setting."